### ISSUE TITLE
Various 'Drag and Drop' functions on Stable Build 0.9.9 no longer working as intended

### ISSUE THREAD

**Issue by Dektarey** [ironmonk88/monks-enhanced-journal#144]
I'd like to admit: I dont know much about coding. I simply made this account to tell you about this issue.

The newest Stable Build 0.9.9, using this module along none others on the DnD system, completely devastated the Drag and Drop functioniality on the Person, Shops, Encounter and Place -Journals. Meaning Relationships on Persons, Items on Shops and so on have no functionality whatsoever with this new patch.

Although the Console still shows that the Drop Event triggers without issues or errors.

I am aware the current version of this module doesnt match the new stable build, but i thought this information might be relevant for you.

And happy holidays.

**Comment by alessiocali** [comment]
I was going to post this as well, I just installed the module for the first time after updating to Foundry v9 and noticed that none of the Drag&Drop functionality worked. I tested this on a clean-slate DnD World just to be sure it wasn't some module conflict.

**Comment by alessiocali** [comment]
I made a quick debugging foray and tried to drop an Actor onto an Encounter page, and ended up inside this function in `enhanced-journal.js`. I saw a similar problem on the `scene-clicker` module, the  `entity` attribute has been deprecated in favor of `document`, not sure after which API version.

![image](https://user-images.githubusercontent.com/7502952/147372610-7084b039-0f8b-4173-ac4a-4dc2af73aee2.png)

**Comment by alessiocali** [comment]
I tracked down the faulty references and I think I fixed them all, I hope. I'll open a pull request soon.

**Comment by Dektarey** [comment]
Indeed. Marvelous work, mate.

I've went through them all once again, just to make sure, and everything's fixed after applying these changes.

**Comment by alessiocali** [comment]
It would be better to not close the issue until the maintainer has merged the changes and they're available for everyone :) Could you open it again? I can't seem to be able to do it myself.

**Comment by Dektarey** [comment]
Oh, i am sorry. As i said, am not very familiar with Github and its procedures.

The changelog says drag & drop has been fixed, so i assume the issue is resolved on the main branch? Should i still re-open this thread nonetheless?

**Comment by alessiocali** [comment]
Disclaimer: I'm not too familiar with the general consensus on GitHub practices but I know something about professional development workflows in general, so this is mostly my opinion on the matter.

I just saw the commit from monk, but yes in general to keep track of what's going on an ideal workflow is the following

* Somebody opens an issue
* Maintainer confirms the issue, works on it
* Maintainer fixes the issue and closes the issue in the process.

Closing can also link the fixing change (a "commit" in git lingo) so you know what fixed what. In general I think an external person should close an issue only if the maintainer is unresponsive and the issue has been fixed already in a later version.
